.github/workflows/release.yaml

```diff
@@ -122,28 +122,6 @@ jobs:
         if: startsWith(env.LAST_COMMIT_MSG, 'ci(release):') && env.RELEASE_VERSION_HIGHER_THAN_LATEST == 'true'
         run: echo SHOULD_DEPLOY_DOCS=true >> $GITHUB_ENV
 
-      - name: Checkout API Docs
```

Comment:
I think if we want to continue with the current workflow, this needs to be re-instated

Reply:
Great catch, done